Clockwise/ICircuitBreakerStorage.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Clockwise
+{
+    public interface ICircuitBreakerStorage
```

Comment:
>ICircuitBreakerStorage [](start = 21, length = 22)

Not sure `storage` is the right name. Isn't this the thing that provides the open/closing policy based on the success/failed votes? Thoughts?

Reply:
ICircuitBreakerStateMachine ?

Reply:
The interface doesn't imply storage or a state machine. Those are implementation details.

How about `ICircuitBreakerBroker`?